### PR TITLE
Add Lum Network resources

### PR DESCRIPTION
**Pull request by lebascou**

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -1517,4 +1524,42 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
...
+			{
+				coinDenom: 'LUM',
+				coinMinimalDenom: 'ulum',
+				coinDecimals: 6,
+				coinGeckoId: 'lum-network',
+				coinImageUrl: window.location.origin + '/public/assets/tokens/lum.svg',
+			},
+		],
+		coinType: 118,
+		features: ['stargate', 'ibc-transfer', 'no-legacy-stdTx', 'ibc-go'],
```
Comment (joeabbey) [inline src/config.ts:1562]: Should there be gas price steps here?

Noticed they are present for $HUAHUA

https://github.com/osmosis-labs/osmosis-frontend/pull/146/files#diff-c3095d5010e65c52737a98a5d618ea24049ebe90c8470752426081d70ed6e012R1560-R1564

**Review by joeabbey — COMMENTED** [review]
Just a comment about gas price steps...

Also, I assume pool creation and configuration also comes soon (given there's a 100 $OSMO cost that comes with that)

File: src/config.ts (hunk 1 above)
Comment (sunnya97) [reply src/config.ts:1562]: Yes

Comment (lebascou) [reply src/config.ts:1562]: Do you need me to do something here? We have a min gas pice set by all validators to `minimum-gas-prices = "0.001ulum"`
Not sure on how to interpret this setting in your file though

Comment (joeabbey) [reply src/config.ts:1562]: Closing the loop, I think it just informs Keplr "reasonable" values for low/medium/high gas

**Pull request merged**